pkg/virt-controller/services/template.go

```diff
@@ -71,10 +71,6 @@ func (t *templateService) RenderLaunchManifest(vm *v1.VirtualMachine) (*kubev1.P
 		Name:      "libvirt-runtime",
 		MountPath: "/var/run/libvirt",
 	})
-	volumesMounts = append(volumesMounts, kubev1.VolumeMount{
```

Comment:
This was needed only for the old emulation, right? Is this needed when the emulation still applies?